### ISSUE TITLE
Can not load videos

### ISSUE THREAD

**Issue by userunknown358** [ftde0/yt2009#144]
I created a private instance of YT2009 and after clicking on a video, I get this message:
"This video is unavailable"

Is that a problem with the API? At this point YT2009 is unusable.

**Comment by ftde0** [comment]
this occurs when the hosting provider has their ip range blocked by youtube, requiring you to use an account to play videos, and in turn, host things like yt2009 from those.

you should have received instructions on how to use an account with yt2009 in the console output when starting yt2009.

**Comment by SuperLuigiKillz04** [comment]
the instructions are in the logs

**Comment by ftde0** [comment]
no response in a while, assuming it's solved? feel free to reopen/create a new issue if still happening
